modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -218,7 +230,7 @@ CWIML4520E: The LDAP operation could not be completed. The LDAP naming exception
 
 To review the expiration time of the Kerberos principal user, run https://docs.oracle.com/en/java/javase/11/tools/klist.html#GUID-205BCE0D-F5D4-4A9C-ACBC-46423BAF616F[the Java klist tool].
 
-For more information, see xref:reference:feature:ldapRegistry-3.0.adoc#krb5[Configure Kerberos authentication for LDAP servers]
+For more information, see xref:reference:feature:ldapRegistry-3.0/examples.adoc#krb5[Configure Kerberos authentication for LDAP servers]
```

Comment:
link:/docs/latest/reference/feature/ldapRegistry-3.0.html#krb5[Configure Kerberos authentication for LDAP servers]